extensions/cornerstone/src/getPanelModule.tsx

```diff
@@ -81,7 +81,7 @@ const getPanelModule = ({ commandsManager, servicesManager, extensionManager }:
       name: 'panelMeasurement',
       iconName: 'tab-linear',
       iconLabel: 'Measure',
-      label: 'Measurement',
+      label: 'All Measurements',
```

Comment:
I think changing this might disrupt other people's tests if they're selecting based on the label. Can we leave this one unchanged?